linker/shared/src/test/scala/org/scalajs/linker/testutils/TestIRBuilder.scala

```diff
@@ -64,4 +84,7 @@ object TestIRBuilder {
 
   def mainModuleInitializers(moduleClassName: String): List[ModuleInitializer] =
     ModuleInitializer.mainMethodWithArgs(moduleClassName, "main") :: Nil
+
+  implicit def string2ident(name: String): Ident =
+    Ident(name)
```

Comment:
If you are introducing this, please replace occurrences of `Ident("...")` (again, possibly in a separate commit).